docs/DeveloperGuide.adoc

```diff
@@ -731,7 +731,49 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
 
+|`* *` |user |view my top five most accessed contact |find them more easily
+
+|`* *` |user |see frequently used contacts instantly |locate them more easily
+
+|`* *` |user |view my contact list online |so that I can view it without the device running the software
+
+|`* *` |user |send email to a group of contacts |easily send email to a group
+
+|`* *` |user |include more details for a person's information |identify the person easily
+
+|`* *` |user |delete multiple users in one command |remove several entries that I no longer need quickly
+
+|`* *` |user |have color-coded commands and attributes in command line |identify the command and attributes easily
+
+|`* *` |user |view my recent additions |identify which contact is newly added
+
+|`* *` |lazy user |have short-hand form of command |perform commands quickly
+
+|`* *` |user |set reminders |remind myself any activities or events that will happen
+
+|`* *` |user |save my contacts as an external file |backup or transfer data easily
+
+|`* *` |user |add multiple emails and phone numbers to a contact |handle contacts with more than one email and phone numbers
+
+|`* *` |user |add birthday of a person |store the birthday of the person
+
+|`* *` |user |add reminders of any approaching birthday |prepare for the person's birthday
```

Comment:
I thought if we allow the adding of birthday information, the program would automatically be able to remind us of any impending birthdays? We need to explicitly add reminders for birthdays?